api/nodes/serializers.py

```diff
@@ -1006,7 +994,15 @@ def to_representation(self, value):
 class NodeContributorsSerializer(JSONAPISerializer):
     """ Separate from UserSerializer due to necessity to override almost every field as read only
     """
-    non_anonymized_fields = ['bibliographic', 'permission']
+    anonymized_fields = [
+        'id',
```

Comment:
This results in `id: ""`, which breaks ember data. Would it be easy to make the entire `[bibliographic_]contributors` relationships 403 for anonymized VOLs? I can also hack the frontend, if need be.

Reply:
Ah, I didn't think of what empty id's would do ember-side, but that totally make sense.  I was just following existing VOL behaviors - we were previously only returning bibliographic and permission info because we didn't want to leak the user id. So 403 NodeBibliographicContributors and NodeContributors relationships for VOL's?

Reply:
Or maybe a simpler solution: Looks like adding `contributors` and `bibliographic_contributors` to `NodeSerializer.anonymized_fields` does the trick nicely. The frontend won't try fetching contributors if it knows the node is anonymized (and it does, thanks to `meta.anonymous`)